### PR TITLE
Fix/endpoint resolver

### PR DESCRIPTION
**Pull request by atierian**
*Issue #, if available:*

*Description of changes:*

*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] All integration tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

*DataStore checkpoints (check when completed)*

- [ ] Ran AWSDataStorePluginIntegrationTests
- [ ] Ran AWSDataStorePluginV2Tests
- [ ] Ran AWSDataStorePluginMultiAuthTests
- [ ] Ran AWSDataStorePluginCPKTests
- [ ] Ran AWSDataStorePluginAuthCognitoTests
- [ ] Ran AWSDataStorePluginAuthIAMTests

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/CustomEndpoint/AWSEndpointResolving.swift [hunk 1]
```diff
@@ -19,8 +24,8 @@ struct AWSEndpointResolving: AWSClientRuntime.EndpointResolver {
         self.endpoint = endpoint
     }
 
-    func resolve(serviceId: String, region: String) throws -> AWSEndpoint {
-        let endpoint = try endpoint()
-        return AWSEndpoint(endpoint: endpoint)
-    }
+//    func resolve(serviceId: String, region: String) throws -> AWSEndpoint {
```
Comment (harsh62) [inline AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/CustomEndpoint/AWSEndpointResolving.swift:27]: Should we remove the commented out code?

Comment (atierian) [reply AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Support/CustomEndpoint/AWSEndpointResolving.swift]: Addressed in [5514065](https://github.com/aws-amplify/amplify-swift/pull/2552/commits/5514065c358ab5fa987c9fd1513bed3d1ffef563)

**Comment by atierian** [comment]
I mentioned this in the original PR of this feature, but going to reiterate it here. We really need to an integration test for this use case to catch any unexpected regressions introduced through updates to the AWS SDK for Swift.

**Review by harsh62 — APPROVED** [review]
LGTM

**Pull request merged**